trinity/plugins/builtin/proc_test/plugin.py

```diff
@@ -0,0 +1,47 @@
+import asyncio
+import logging
+from typing import (
+    Any,
+    Dict,
+)
+from lahja import (
+    Endpoint
+)
+
+from trinity.events import (
+    PeerCountRequest,
+)
+from trinity.extensibility import (
+    BaseOwnProcessPlugin,
+)
+
+
+async def request_receive_peer_count(event_bus: Endpoint) -> None:
+    while True:
+        response = await event_bus.request(PeerCountRequest())
+        print("Peer Count: " + str(response.payload))
```

Comment:
Can this be a logging statement?  This also implies that we need to have the plugin API correctly configure logging to go over the queued logging API.

Reply:
Yes. In fact, the queued logging is working for the isolated plugins already. This is really just dummy code. I have half of the work done to move the JSON-RPC server into a plugin that runs in an isolated process and communicates via event bus with the peer pool. I think that will be the first useful plugin.